docs/en/stack/ml/nlp/ml-nlp-shared.asciidoc

```diff
@@ -1,3 +1,22 @@
+tag::ml-nlp-adaptive-allocations[]
+The numbers of threads and allocations you can set manually for a model remain constant even when not all the available resources are fully used or when the load on the model requires more resources.
+Adaptive allocations, however, provide full control and flexibility over the model allocations, and help you to manage performance and cost more easily.
```

Comment:
Not so sure about this "full control and flexibility": you actually give away control of the number of allocations.

What about: "Instead of setting the number of allocations manually, you can enable adaptive allocations to set the number of allocations based on the load on the process. This can help you to manage performance and cost more easily."